custom_commands/suppressedNotificationsTrueCheck.js

```diff
@@ -0,0 +1,70 @@
+const axios = require('axios');
+
+module.exports = {
+    command: async function () {
+
+        const getToken = async () => {
+            const response = await axios.post(
+                'https://qa2.development.tidepool.org/auth/login',
+                '',
+                {
+                    headers: {
+                        'Authorization': 'Basic YnJpYW4rdXBsb2FkNzg2d2ViMTMyN2NsaW5pYzM1YUB0aWRlcG9vbC5vcmc6dGlkZXBvb2w=',
+                        'Content-Type': 'application/x-www-form-urlencoded'
+                    }
+                })
+            return response.headers['x-tidepool-session-token']
+        }
+
+
+        const clinicData = async () => {
+            
+            const token = await getToken();
+            response = await axios.get('https://qa2.development.tidepool.org/v1/clinicians/4ae71760b6/clinics', {
```

Comment:
Typically, we should inject environment-specific variables like the URL (or credentials) from environment variables, injected by the CI at run-time.

Reply:
Yeah, with that, do we need to create some config files for each environment?
You can check, that in this PR, we use clinic ID in URL (`https://qa2.development.tidepool.org/v1/clinicians/4ae71760b6/clinics`) and have this condition - 
```js
if (data[i].clinic.id == '62419f38f85189a39ac4b68d') {
```

So it would be 
```js
const data = {
  prod: {
    clinicId: "123",
    specificClinicId: "456",
  },
  qa1: {
    clinicId: "789",
    specificClinicId: "012",
  },
  qa2: {
    clinicId: "4ae71760b6",
    specificClinicId: "62419f38f85189a39ac4b68d",
  },
  ...
};
```

Or should we mock it? 🤔

Reply:
Yeah, I am sorry, I am definitely missing some context here for sure, so I'd be happy to get more context.

Reply:
@vojtech-cerveny clinicId and clinicianId are moved to environment global variables

see latest commit e138f9274a6bf9b2d17620d2a10426b1651c4fb2